test/e2e/autotls/auto_tls_test.go

```diff
@@ -91,6 +93,41 @@ func testAutoTLS(t *testing.T) {
 	rootCAs := createRootCAs(t, clients, objects.Route.Namespace, certName)
 	httpsClient := createHTTPSClient(t, clients, objects, rootCAs)
 	testingress.RuntimeRequest(t, httpsClient, "https://"+objects.Service.Status.URL.Host)
+
+	t.Run("Tag route", func(t *testing.T) {
+		// Verify that a certificate is created when route is tagged
+		if _, err := v1test.UpdateServiceRouteSpec(t, clients, names, servingv1.RouteSpec{
+			Traffic: []servingv1.TrafficTarget{{
+				Tag:            "tag1",
+				Percent:        ptr.Int64(100),
+				LatestRevision: ptr.Bool(true),
+			}},
+		}); err != nil {
+			t.Fatalf("Failed to update Service route spec: %v", err)
+		}
+		// Wait for ingress TLS configuration.
+		if err = v1test.WaitForServiceState(clients.ServingClient, names.Service, v1test.IsServiceReady, "ServiceIsReady"); err != nil {
```

Comment:
Are we able to check if the URL of target traffic (https://github.com/knative/serving/blob/master/pkg/apis/serving/v1/route_types.go#L107) becomes to HTTPS here?

Reply:
Do you mean instead of waiting for the service to be ready with `IsServiceReady` we should wait for the URL to be HTTPS?

Reply:
Oh, I see how it's done above. I'll use `httpsReady`